src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -20,25 +23,45 @@
     public static Student[] getSampleStudents() {
         return new Student[] {
             new Student(new Name("Alex Yeoh"), new Telegram("41ex_Yo"), new Email("alexyeoh@example.com"),
-                getTagSet("Average")),
+                getTagSet("Average", "CS2103T Tutorial", "CS2100 Lab")),
             new Student(new Name("Bernice Yu"), new Telegram("b3rnice"), new Email("berniceyu@example.com"),
-                getTagSet("Good", "Experienced")),
+                getTagSet("Good", "Experienced", "CS2103T Tutorial")),
             new Student(new Name("Charlotte Oliveiro"), new Telegram("C_Ol1ve"), new Email("charlotte@example.com"),
-                getTagSet("Struggling")),
+                getTagSet("Struggling", "CS2103T Tutorial")),
             new Student(new Name("David Li"), new Telegram("li_DAvid"), new Email("lidavid@example.com"),
-                getTagSet("Weak")),
+                getTagSet("Weak", "CS2100 Lab")),
```

Comment:
I would advise against adding the class names as `Tags` since it might confuse our first time users into thinking that linking students to classes would automatically add the classes as `Tags`.